src/Services/EncounterService.php

```diff
@@ -311,6 +313,60 @@ public function insertEncounter($pid, $data)
             return $results;
         }
     }
+
+    public function updateEncounter($pid, $eid, $data)
+    {
+
```

Comment:
travis/psr2 will fail empty lines at top of functions

Reply:
Done